components/clarity-vscode/package.json

```diff
@@ -23,20 +26,33 @@
     "dev:watch": "webpack -c ./webpack.config.dev.js -w",
     "dev:browser": "vscode-test-web --extensionDevelopmentPath=. ./test-data --open-devtools --browser=chromium",
     "dev": "webpack -c ./webpack.config.dev.js; concurrently \"npm:dev:*\"",
-    "package": "npm run clean; NODE_ENV=production webpack",
-    "vsce:package": "npm run package; vsce package",
-    "vsce:publish": "npm run package; vsce publish"
+    "vscode:prepublish": "npm run clean; NODE_ENV=production webpack",
+    "vsce:package": "vsce package",
+    "vsce:publish": "vsce publish",
+    "vsce:prerelease": "vsce publish --pre-release"
   },
   "categories": [
     "Programming Languages",
+    "Linters",
     "Debuggers"
   ],
   "keywords": [
+    "blockchain",
+    "clarity",
+    "bitcoin",
+    "stacks",
+    "smart contract",
     "lsp",
-    "web"
+    "stx",
+    "btc"
   ],
+  "galleryBanner": {
+    "color": "#242424",
+    "theme": "dark"
+  },
+  "qna": "https://docs.hiro.so/",
   "engines": {
-    "vscode": "^1.43.0"
+    "vscode": "^1.64.0"
```

Comment:
>=1.63 is required to use [pre-release](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prerelease-extensions) which i think I will.
(I went for 1.64 which was release in January 2022. Didn't see the point of putting super old version)